.circleci/config.yml

```diff
@@ -478,6 +479,20 @@ commands:
           command: |
             gcloud container clusters delete <<parameters.cluster-id>> --quiet --region=<<parameters.region>>
 
+  terminate-gke-cluster-cuda-11:
+    parameters:
+      cluster-id:
+        type: string
+      zone:
+        type: string
+    steps:
+      # Use a run instead of `gke/delete-cluster` because circle CI orbs do not support `when`.
```

Comment:
I'm confused by this comment... isn't there a `when` right there?

Reply:
IIRC I'm referring to the delete-cluster command in the gke orb. I don't think I could use that and tell it to always run, even after failure, but I could with just running the CLI command directly. Would be happy if I was wrong - I originally wrote this months ago when I was even newer to CircleCI than I am right now...